ultra/docs/agent.md

```diff
@@ -0,0 +1,40 @@
+# ULTRA Agents
+
+An ULTRA agent specification inherits from SMARTS's `AgentSpec`. The baseline agents of ULTRA use the `BaslineAgentSpec` available in `ultra/baselines/agent_spec.py`. However, the following are the attributes of an `AgentSpec` available for customization:  
+* agent interface
+* agent parameters
+* agent policy class
+* observation adapter
+* reward adapter
+* info adapter
+* action adapter
```

Comment:
This list seems redundant given that each is elaborated on after the code block.
```suggestion
```

Reply:
Oh okay yeah, just noticed that too. I will modify this, thanks!